src/base/emu-i386/simx86/cpu-emu.c

```diff
@@ -1142,6 +1144,31 @@ int e_dpmi(cpuctx_t *scp)
   return retval;
 }
 
+/* set special SIM mode for VGAEMU faults */
+int instr_emu_sim(cpuctx_t *scp, int pmode, int cnt)
+{
+  instr_emu_sim_reset_count(cnt);
+#ifdef HOST_ARCH_X86
+  int orig_sim = config.cpusim;
+  if (!CONFIG_CPUSIM) {
+    config.cpusim = 1;
```

Comment:
Same here, maybe not to touch config
var, but change the macro?

Reply:
I've now changed the macro, no need for a function since with #1982 it's internal to simx86.